### ISSUE TITLE
Support separate S3 bucket for jsPsych videos

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1442]
# TL;DR

Video recordings from jsPsych studies will be stored in a separate AWS S3 bucket. We need to set up the lookit-api and add credentials to retrieve videos from jsPsych studies from the correct bucket.

# Summary

The lookit-api will need to access videos from jsPsych studies from the correct S3 bucket, and with appropriate credentials. This is relevant to:
1. the video response download/view options for researchers (`StudyResponseVideoAttachment`, `StudyAttachments`, `build_zipfile_of_videos` task)
2. the family's study history page, where the family can view their videos from previous study sessions (`StudiesHistoryView`)
 
Currently the lookit-api checks the `recording_method` to determine which S3 bucket to use when generating a download/view URL (see [these lines](https://github.com/lookit/lookit-api/blob/a75fe88768eec65ee6424ffcdafcb1e5a8b20682/attachment_helpers.py#L22-L48)). We will need to keep doing that for EFP studies (study type 1), and use a different bucket for jsPsych studies (study type 3).
